### ISSUE TITLE
Can't run in docker?

### ISSUE THREAD

**Issue by dkegel-sigsci** [alpinelinux/alpine-make-vm-image#15]
Running in an alpine linux docker container on mac fails with
```
> Attaching image foo.img as a NBD device
modprobe: can't change directory to '/lib/modules': No such file or directory
ERROR: No available nbd device found!
```

nbd seems like overkill when creating raw images; using a loopback mount for the raw image case
might let the script run in docker.

**Comment by pojntfx** [comment]
Running with `--privileged` is a quick and dirty solution.

**Comment by pojntfx** [comment]
@dkegel-sigsci Just wrote a small Go tool to use this script with Docker/build custom Alpine Images in Docker if you're interested: https://github.com/pojntfx/alpimager

**Comment by dkegel-sigsci** [comment]
Thanks to both.  --privileged is enough for me for now!

**Comment by dkegel-sigsci** [comment]
Duplicate of #4 ?

**Comment by rhpk** [comment]
--privileged worked fine for me but don't forget to modproble nbd before starting the script.
